examples/acquiring_data/downloading_hmi.py

```diff
@@ -32,15 +35,26 @@
 # don't provide a path it will download the file into the sunpy data directory.
 # The output provides the path of the downloaded files.
 # The result can be from several data clients, so we have to index the client first
-# client and then index the file.
+# and then index the file.
 
 # Slice the first record returned by the first client.
-downloaded_file = Fido.fetch(result[0, 0])
```

Comment:
I think the slicing syntax that you've removed here is actually correct. Indexing it as `[0]` just returns the same response so this will still download more than 1 file if your query returns more than one file. However, it seems like `[0,0]` should not work either given #2781?

Reply:
Yes, `[0,0]` returns an error, so I kept it the following line:  
`downloaded_file = Fido.fetch(result[0])`. 
 
 Because the search only spans a few minutes, executing this line only downloads one file.